crates/release_plz/tests/all/release_pr.rs

```diff
@@ -10,6 +10,14 @@ async fn release_plz_should_set_custom_pr_details() {
     let config = r#"
     [workspace]
     pr_name = "release: {{ package }} {{ version }}"
+    pr_body = """
+    {% for release in releases %}
+    === {{release.title}}
```

Comment:
Let's use markdown headers in our examples and tests (`#` instead of `=`) so that the changelog parser works 👍